### ISSUE TITLE
Prediction with best weights get error

### ISSUE THREAD

**Issue by mdziec** [sassoftware/python-dlpy#91]
After saving best weights with save_best_weights=True (TinyYoloV2 model) option I try to use this weights and predict.

model.set_weights('model_best_weights_uvSn8E')
model.predict(data=dettbl) #dettbl is training table

I get error message:

NOTE: Model weights attached successfully!
ERROR: A floating point exception or access violation occurred that prevents progress of the analysis. This is usually caused by wrong scales of inputs, large learning rate, or exploding gradients.
ERROR: The action stopped due to errors.

**Comment by sasxianhu** [comment]
what happens if you do not use the bestweights?  are you able to get anything?

**Comment by mdziec** [comment]
If I don’t use best weights in “set_weight” prediction work normally, without errors.

**Comment by wenyushi** [comment]
Hi, I cannot repeat the error. Could you send me your script? So that I can debug it. Thanks.

**Comment by mdziec** [comment]
[detekcja_obiektow_test.zip](https://github.com/sassoftware/python-dlpy/files/2688036/detekcja_obiektow_test.zip)

**Comment by mdziec** [comment]
I use MPP but without GPU. I will try to do a test on SMP.

**Comment by ethem-kinginthenorth** [comment]
this should have been fixed with the new vdmml release. I am closing this issue and please re-open it if it happens with the newest vdmml release. thanks
